bears/css/PostCSSBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Setting import typed_list
+
+
+class PostCSSBear(Lint, LocalBear):
+    executable = "postcss"
+    diff_message = "PostCSS plugins in action."
```

Comment:
yes the message needs to be better, maybe we can make some custom ones for the different plugins?